src/widgets.rs

```diff
@@ -1,6 +1,6 @@
 //! `widgets` is a collection of types that implement [`Widget`] or [`StatefulWidget`] or both.
 //!
-//! All widgets are implemented using the builder pattern and are consumable objects. They are not
+//! All widgets are implemented by the fluent interface and are consumable objects. They are not
```

Comment:
I think we can drop this wording entirely - it's not particularly relevant.

```suggestion
//! Widgets are created for each frame as they are consumed by when rendered. They are not
```

Perhaps as a second sentence "Most widgets implement their fluent style setter methods like `fn(mut self, value: ...) -> Self`. The results of each setter call must be chained with another call or used.

You'll probably need to reformat the code locally if you take this suggestion (so don't just merge it from github)